shared/js/background/dnr-session-rule-id.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * For managing dynamically created MV3 session rules
+ * getNextSessionRuleId will return the next unique session rule id to use when creating new session rules
+ **/
+const browserWrapper = require('./wrapper.es6')
+const SESSION_RULE_ID_START = 100000
```

Comment:
Please can you exportt `SESSION_RULE_ID_START` from the declarative-net-request module and add a comment there to explain?

Reply:
Let's tackle this in a small follow up commit and land this for now.